src/Sylius/Behat/Context/Api/Common/SaveContext.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Common;
```

Comment:
Not sure if this additional namespace is necessary here:
```suggestion
namespace Sylius\Behat\Context\Api;
```